ui/app/AppLayouts/Onboarding2/stores/OnboardingStore.qml

```diff
@@ -7,17 +7,26 @@ import AppLayouts.Onboarding.enums 1.0
 QtObject {
     id: root
 
-    signal appLoaded()
+    signal appLoaded
+    signal setPinFailed(string error)
+    signal authorizationFailed(string error)
+    signal keycardExportKeysSuccess
+    signal keycardExportKeysFailed(string error)
 
     readonly property QtObject d: StatusQUtils.QObject {
         id: d
         readonly property var onboardingModuleInst: onboardingModule
 
         Component.onCompleted: {
-            onboardingModuleInst.appLoaded.connect(root.appLoaded)
-            onboardingModuleInst.accountLoginError.connect(root.accountLoginError)
-            onboardingModuleInst.obtainingPasswordSuccess.connect(root.obtainingPasswordSuccess)
-            onboardingModuleInst.obtainingPasswordError.connect(root.obtainingPasswordError)
+            d.onboardingModuleInst.appLoaded.connect(root.appLoaded)
+            // TODO implement the following signals
```

Comment:
Even if not implemented yet, this can stay here uncommented, right?

Reply:
Since the view functions don't exist, any `connect` put below those don't activate because the function throws an error.
It felt safer to comment those for now as it confused me for a while why my signals didn't work.

Reply:
Just declare those signals?

Reply:
@igor-sirotin I'd just declare the signals in NIM, even w/o actually using them at this point; then we can leave this block uncommented

Reply:
@caybro I added `accountLoginError` and removed the biometrics signals connections, as they won'y be coming from Nim.